monster.cpp

```diff
@@ -1,11 +1,13 @@
 #include <iostream>
 #include "monster.h"
+#include "rng.h"
 
 using namespace std;
 
-Monster::Monster(int health_in, int attack_in, int defense_in, int specialAttack_in, int specialDefense_in, int speed_in, string name_in) :
-	name(name_in), experience(0), level(1), maxHealth(health_in), currentHealth(health_in), attack(attack_in), defense(defense_in), specialAttack(specialAttack_in), specialDefense(specialDefense_in), speed(speed_in)
-{}
+Monster::Monster(int health_in, int attack_in, int defense_in, int specialAttack_in, int specialDefense_in, int speed_in, string name_in) : name(name_in), experience(0), level(1), maxHealth(health_in), currentHealth(health_in), attack(attack_in), defense(defense_in), specialAttack(specialAttack_in), specialDefense(specialDefense_in), speed(speed_in)
+{
+}
+Random randomizer;
```

Comment:
This is a global instantiation, kinda bad to do in C++ (which is why you had to name the other one different). I would say instead to instantiate it within the Monster class.